boards/arm/96b_argonkey/Kconfig.defconfig

```diff
@@ -71,6 +71,12 @@ endif # I2S
 
 if DMA
 
+config DMA_LOG_LEVEL
```

Comment:
we should not be setting LOG level in the board defconfig, this is up to the application to set the log level, not the board.

Reply:
OK, I moved both of the in the prj.conf file of the sample code for the microphone. I will delete this PR if you agree.